api/src/main/java/org/motechproject/nms/api/web/CallDetailsController.java

```diff
@@ -186,13 +193,31 @@ private String validateCallDetailsCommonElements(CallDetailRecordRequest callDet
     private String validateCallDetailsMobileKunjiElements(CallDetailRecordRequest callDetailRecordRequest) {
         StringBuilder failureReasons = new StringBuilder();
 
-        if (null == callDetailRecordRequest.getWelcomeMessagePromptFlag()) {
+        if (callDetailRecordRequest.getWelcomeMessagePromptFlag() == null) {
             failureReasons.append(String.format(NOT_PRESENT, "welcomeMessagePromptFlag"));
         }
 
         return failureReasons.toString();
     }
 
+    private String validateCallDetailsMobileAcademyElements(CallDetailRecordRequest callDetailRecordRequest) {
+        StringBuilder failureReasons = new StringBuilder();
+
+
+
```

Comment:
Three blank lines?  Really?

![really-seth-and-amy](https://cloud.githubusercontent.com/assets/51527/8788585/3f561b26-2ef0-11e5-88fe-e1c63d8a9d8d.jpeg)